src/routes.php

```diff
@@ -14,4 +14,6 @@
 
 $app->get('/admin', 'adminControllerFactory');
 
-// API routes:
\ No newline at end of file
+$app->get('/register', 'registerControllerFactory');
```

Comment:
I think we should be calling the 'registerController' here, and in dependencies.php make registerController point at the factory